services/wireguard/endpoint/proxyclient/client_test.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
```

Comment:
For new files, this should be set to the current date. 
```suggestion
 * Copyright (C) 2022 The "MysteriumNetwork/node" Authors.
```